_posts/2024-05-05-breaking-cultural-barriers.html

```diff
@@ -0,0 +1,36 @@
+    <p>However, after months of trying with no success, I eventually made the difficult decision to let go of my tech career aspirations, particularly as I found out I was pregnant once again. As an Asian wife, societal norms emphasized my role in managing the household, caring for the children, and attending to domestic responsibilities. Balancing these duties left me with little energy to ponder my future or career path.</p>
+    <p>With a blink of an eye and fast forward my life 14 years, I had now 4 kids and suddenly, everything seemed to revolve around the needs of my growing family. The dreams and aspirations I once held dear faded into the background as I embraced the role of motherhood and wife.</p>
+    <p>Slowly but surely, I began to lose sight of who I was beneath the layers of responsibility. It was like I was on autopilot, just going through the motion’s day in and day out. The spark of ambition that once burned brightly within me was dimmed by the constant reminder that my primary duty was to be the perfect caregiver and homemaker.</p>
+    <p>In conforming to these societal expectations, I found myself drifting further away from my true self. My career in tech to be seemed like a distant memory, overshadowed by the demands of family life.</p>
+    <p>There was a time when I felt lost and unsure of myself. It was like I was stuck in the dark, unable to find my way. But then, my eldest brother came to my rescue. He saw the struggles I was going through and made it his mission to help me find my way back to the light. With his support, I started to feel hopeful again.</p>
+    <img class="img-fluid" src="/assets/images/blog/breaking-barriers/quote1.png" alt="quote1">
+    <p>I also started to think differently and stumbled upon podcasts where women had made differences in tech/coding and self-help books, which slowly shifted my perspective. Feeling more aware, I made small changes like going for walks, joining a gym, and rediscovering the joy of riding my bike from childhood. These simple actions brought renewal and inspiration to my life, showing me the power of embracing change and finding joy in everyday experiences.</p>
+    <p>The more I heard about women making their mark in tech, the stronger my sense of determination grew and then I stumbled upon inspiring stories on social media about mothers and older women changing careers. It got me thinking about my own path, so I started looking into courses.</p>
+    <img class="img-fluid" src="/assets/images/blog/breaking-barriers/quote2.png" alt="quote2">
```

Comment:
I think after each image is necessary to add some space, it can be a new line 🙈  or 
better use css class to add margin bottom.

Reply:
Good point! I added css class with margin :)